edx-olive/configuration-old
# SE-1671 Add newrelic monitoring for mongo

This PR adds a playbook to deploy the newrelic infrastructure agent, mongo monitoring plugin, and configuration for monitoring the mongo cluster. This playbook is designed to be run against each of the mongo nodes.


**Test instructions**:

- run the newrelic_mongo_monitor.yml playbook against an existing mongo node
- check the newrelic thirdparty integrations for mongo dashboard and verify that it is monitoring the node just deployed.

**Author notes**:

- also fixes an issue where playbooks can't be run when they are in the `playbooks/olive` directory. This moves them to the parent directory.

**Reviewer**:
